### PR TITLE
Add .travis.yml for basic CI

### PR DESCRIPTION
**Pull request by fabiand**
This PR does the following:

- Add .travis.yml
- Hook this repo up to travis
- Setup minikube
- Deploy KubeVirt
- Deploy CDI

Signed-off-by: Fabian Deutsch <fabiand@fedoraproject.org>

### CODE REVIEW

**Comment by copejon** [comment]
Thanks! LGTM

Side question - where will the file server for `$SRC` live?   Is that something the CI will spin up?

**Comment by fabiand** [comment]
In this first iteration it will just be a remote image (Fedora image from the internet) in future this can also be a local image.

**Comment by fabiand** [comment]
Oh, and you are welcome, looking forward to see if this is going to work out :)

**Comment by fabiand** [comment]
/me notes that he just has issues to get travis configured, it's not picking up this PR

**Comment by fabiand** [comment]
Yey. it was a syntax error. let's see what CI will say.

**Comment by copejon** [comment]
Thinking on image size - can we use something like TinyCore to reduce the CI runtime?

**Comment by fabiand** [comment]
Yep. I'll use that image instead.

**Comment by fabiand** [comment]
So - The PR is ready.

but it's just a start.

I need help to create - in a separate PR I suppose - the manifests which are required to test CDI.

File: Makefile [hunk 1]
```diff
@@ -83,6 +83,10 @@ release:
 	docker tag $(CONTROLLER_IMAGE) $(REGISTRY)/$(CONTROLLER_IMAGE):latest
 	docker push $(REGISTRY)/$(CONTROLLER_IMAGE):latest
 
+manifests/golden-pvc.yaml: manifests/golden-pvc.yaml.in
```
Comment (copejon) [inline Makefile]: can you move this to the end?  The ugly, commented out code block blow is _eventually_ supposed to be part of `release:` :)

**Comment by copejon** [comment]
@fabiand Can you revert this the `golden-pvc.yaml.in` filename back to it's original?

**Comment by fabiand** [comment]
Done

**Pull request merged**